api/optimeet/groups/views.py

```diff
@@ -5,22 +5,31 @@
 from django.views.decorators.csrf import csrf_exempt
 from rest_framework.parsers import JSONParser
 from rest_framework import status
+from django import db
 
+from .models import rnd_id
 from . import models
 from . import serializers
 
 @csrf_exempt
 @api_view(['POST'])
 def groups(request):
     
-    serializer = serializers.GroupSerializer(data = request.data)
+    get_id = rnd_id()
+
+    request_data = request.data.copy()
+    id = {'group_id':get_id}
+    request_data.update(id)
+
+    serializer = serializers.GroupSerializer(data = request_data)
```

Comment:
I think doing it through the view would be a bit cleaner, wouldn't have to modify the incoming data before it gets validated:
```
def create(self, validated_data):
    validated_data['group_id'] = rnd_id()
    return super(GroupSerializer, self).create(validated_data)
```

Reply:
we'll need to pass something for group_id when posting the group. the function will change it later. Is that ok?

Reply:
done